src/tests/integration/sources.rs

```diff
@@ -76,8 +76,8 @@ async fn test_add_source_with_invalid_url() {
 }
 
 #[actix_web::test]
-async fn test_add_source_with_invalid_name() {
-    let auth_id = "test_add_source_with_invalid_url".to_string();
+async fn when_add_source_with_invalid_name_expect_error() {
```

Comment:
Should be be expecting a BadFormat error here? `when_add_source_with_invalid_url_expect_bad_format`?